zerver/lib/bulk_create.py

```diff
@@ -67,13 +67,13 @@ def bulk_create_users(realms, users_raw, bot_type=None, tos_version=None):
     Subscription.objects.bulk_create(subscriptions_to_create)
 
 def bulk_create_streams(realms, stream_list):
-    # type: (Mapping[text_type, Realm], Iterable[Tuple[text_type, text_type]]) -> None
+    # type: (Mapping[text_type, Realm], Set[Tuple[text_type, str, text_type, bool]]) -> None
```

Comment:
Might also make this a dict like in
 zilencer/management/commands/populate_db.py